styles.css

```diff
@@ -102,3 +153,138 @@
 .fa:hover {
   color: #36b37e;
 }
+
+.grid-container {
+  padding-top: 50px;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Good job changing the job using the `display: grid` property on your works section. but let's drop the `flex-direction` property since this is a property that only works in a `flex` element. Instead, you can set your grid to have one column with the grid exclusive properties. [You can find more on this subject here.](https://css-tricks.com/snippets/css/complete-guide-grid/)